modules/offlinemaps/src/main/java/com/sygic/maps/offlinemaps/regions/RegionsViewModel.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2020 Sygic a.s. All rights reserved.
+ *
+ * This project is licensed under the MIT License.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.sygic.maps.offlinemaps.regions
+
+import com.sygic.maps.offlinemaps.base.NavigationViewModel
+import com.sygic.maps.offlinemaps.extensions.RegionAdapterHandler
+import com.sygic.maps.offlinemaps.loader.CountryHolder
+import com.sygic.maps.offlinemaps.loader.MapLoaderGlobal
+import com.sygic.maps.offlinemaps.loader.RegionHolder
+
+class RegionsViewModel : NavigationViewModel(), RegionAdapterHandler {
+    var country = ""
+    var installed = false
+
+    var countryHolder: CountryHolder? = null
+
+    val countryObservable by lazy(LazyThreadSafetyMode.NONE) {
```

Comment:
pls preco pouzivame `LazyThreadSafetyMode.NONE` ?

Reply:
Chcel som sa vyhnut zbytocnej synchronizacii kedze k tomu pristupujem iba z jedneho vlakna, ale uz som to dal prec.